### ISSUE TITLE
Implement an import option in `rvc`

### ISSUE THREAD

**Issue by erikbor** [riboseinc/rvc#95]
@jjr840430 can you implement an import option in `rvc` please?

The import option will do the following:

```console
$ sudo rvc connection new-from-tblk [vpn-name] [/path/to/client.tblk]
$ sudo rvc connection new-from-ovpn [vpn-name] [/path/to/foobar.ovpn]
```

These are (minimum) steps that `rvc` needs to do when the `import` option is given to safely be ran with sudo:
* `rvc` sees the `import` option
* `rvc` checks its own path and verifies it's started from `/opt/rvc/bin/rvc`
* `rvc` `geteuid()` check for root
* `rvc` checks for the `SUDO_UID` environment variable to see if it's started by `sudo`
* `rvc` checks whether the `.ovpn` file is a regular file and not a symlink (please follow the CERT secure coding guideline [Avoid race conditions while checking for the existence of a symbolic link](https://www.securecoding.cert.org/confluence/display/c/POS35-C.+Avoid+race+conditions+while+checking+for+the+existence+of+a+symbolic+link)
* `rvc` checks whether the `.ovpn` file is owned by the UID in the `SUDO_UID` environment variable
* `rvc` checks for the `.ovpn` file extension
* `rvc` copies the `.ovpn` file into `/opt/rvc/etc/vpn.d` with a maximum of 10 kilobytes (the maximum size of a valid `.ovpn` file), the correct permissions will be applied during opening for writing (`chmod 600`), and the `/opt/rvc/etc/vpn.d/new-vpn.ovpn` file is owned by `root:wheel`
* `rvc` drops its privilege to `SUDO_UID` and sends the `reload` command to `rvd` socket

**Comment by jjr840430** [comment]
@erikbor Here is the command line to import VPN configuration.

`sudo rvc import <new-from-tblk|new-from-ovpn> <path>`

I think VPN connection name isn't needed because tblk or ovpn file contains that name.
Let me know your opinions.

**Comment by jjr840430** [comment]
@erikbor Since imported VPN connections should be located into `/opt/rvc/etc/vpn.d`, I think this path should be added by default when initializing rvd.
So, all imported VPN connections should be available even if the rvd global configuration doesn't have this path

**Comment by jjr840430** [comment]
@erikbor What's the option if same VPN configuration exists in `/opt/rvc/etc/vpn.d`?

**Comment by ronaldtse** [comment]
@jjr840430 agree that we should initialize it or install it in the beginning, and we don't need the name.

**Comment by jjr840430** [comment]
@erikbor What's mean of tblk extension? Is it same with .json?

**Comment by ronaldtse** [comment]
"tlbk" is the Tunnelblick extension on macOS, it is basically a directory that contains an '.ovpn' file, a 'ca.crt', a 'client.crt' and a 'client.key' file, all under a 'Resources/Contents' directory.

**Comment by ronaldtse** [comment]
If you install Tunnelblick and "import" an '.ovpn' file, you can see where it is installed, it will be a '.tblk' file.

**Comment by jjr840430** [comment]
Implemented by #112
